Makefile

```diff
@@ -18,8 +18,9 @@ endif
 include $(SHIPYARD_DIR)/Makefile.inc
 
 TARGETS := $(shell ls -p scripts | grep -v -e /)
-export LDFLAGS = -X main.VERSION=$(VERSION)
-
+GIT_COMMIT_HASH := $(shell git show -s --format='format:%H')
+GIT_COMMIT_DATE := $(shell git show -s --format='format:%cI')
+export LDFLAGS =-X github.com/submariner-io/submariner/pkg/version.version=$(VERSION) -X github.com/submariner-io/submariner/pkg/version.gitCommitHash=$(GIT_COMMIT_HASH) -X github.com/submariner-io/submariner/pkg/version.gitCommitDate=$(GIT_COMMIT_DATE)
```

Comment:
```suggestion
export LDFLAGS =-X github.com/submariner-io/submariner/pkg/versions.version=$(VERSION) -X github.com/submariner-io/submariner/pkg/versions.gitCommitHash=$(GIT_COMMIT_HASH) -X github.com/submariner-io/submariner/pkg/versions.gitCommitDate=$(GIT_COMMIT_DATE)
```

Reply:
If we touch this it would be nice to avoid the duplication of `github.com/submariner-io/submariner/pkg/versions` and the long line:

    VERSIONS_MODULE = github.com/submariner-io/submariner/pkg/versions

    export LDFLAGS = -X $(VERSIONS_MODULE).version=$(VERSION) \
                     -X $(VERSIONS_MODULE).gitCommitHash=$(GIT_COMMIT_HASH) \
                     -X $(VERSIONS_MODULE).gitCommitDate=$(GIT_COMMIT_DATE)

Reply:
I'm sorry for the nits, Done.